src/test/java/com/google/cloud/spanner/jdbc/JdbcDatabaseMetaDataWithMockedServerTest.java

```diff
@@ -66,8 +72,23 @@
           .setMetadata(RESULTSET_METADATA)
           .build();
 
+  private static final String GSQL_STATEMENT = "/*GSQL*/";
```

Comment:
We seem to be relying on the first line starting with `/*GSQL*/`. Are there any other possible approaches for differentiating? Example: different directory structure.

Reply:
No, this is actually used by the backend to determine which query engine to use for the query, so we mimic it here. The metadata queries are defined only once, and they are all defined in standard dialect. The backend will also look for this header, and if found, use the Google standard dialect for the query even if the database is in PG dialect.